packages/terra-image/src/Image.jsx

```diff
@@ -29,6 +29,14 @@ const propTypes = {
    * Sets the width of the image.
    */
   width: PropTypes.string,
+  /**
+   * Function to be executed when the image load is successful.
+   */
+  onLoad: PropTypes.func,
```

Comment:
Do we need default props for theses functions? We have other func props that set the value to undefined, such onChange for terra-form.